blackbox/dryrun.py

```diff
@@ -0,0 +1,919 @@
+import base64
```

Comment:
With this relatively old (and slightly refactored code) coming from PySDK, we should consider removing -or at least deprecating- [PySDK's testing folder](https://github.com/algorand/py-algorand-sdk/tree/develop/algosdk/testing). This python functionality is definitely not reproduced in our other SDK's.

@jasonpaulos @winder

Reply:
@barnjamin is introducing `algosdk/dryrun_results.py` which is a separate module outsidef of  `algosdk/testing/`. It's reasonable that the code living in `algosdk/testing/` would be refactored to use the code now being introduced, but the bigger question is it should even exist in PySDK?